tests/test_paillier.py

```diff
@@ -122,3 +122,33 @@ def testScalar(self):
 
         x *= 2
         self.assertTrue(s.decrypt(x) == np.array([2., 4., 6., 8., 10.]))
+
+
+class DivTests(unittest.TestCase):
+
+    def testBasic(self):
+        p, s = KeyPair().generate()
+
+        x = PaillierTensor(p, np.array([3., 8., 15., 24., 35.]))
+        x2 = TensorBase(np.array([3, 4, 5, 6, 7.]))
+
+        y = x / x2
+        print(y.decrypt(s))
+        self.assertTrue(y.decrypt(s) == np.array([1., 2., 3., 4., 5.]))
+
+    def testInline(self):
```

Comment:
isn't that identical to `testBasic` except how the results are stored?

Reply:
yeah... for some functions its made a difference though. in this case i didn't create a separate "inline" function.

Reply:
i just took the relevant ones for multiplication and changed them for addition